pkg/vector/build.yml

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2025 Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+image: eve-vector
+org: lfedge
+network: yes
+config:
+  binds:
+    - /run:/run
+    - /persist:/persist
+  net: new
```

Comment:
Ah, ok, I see it's isolated now... So, no access to it from the host or to the host.

Reply:
It's only accessible from inside vector's container.